experimental/dds/sequence-deprecated/src/sharedIntervalCollection.ts

```diff
@@ -76,11 +78,6 @@ export class SharedIntervalCollectionFactory implements IChannelFactory {
     }
 }
 
-export interface ISharedIntervalCollection<TInterval extends ISerializableInterval> {
```

Comment:
let's not move this in this pr, as i think this is what is force us to also break default map into a new package